test/parallel/test-readline-promises-interface.js

```diff
@@ -910,6 +910,21 @@ for (let i = 0; i < 12; i++) {
     rli.close();
   }
 
+  // Throw an error when question is executed with an aborted signal
+  {
+    const ac = new AbortController();
+    const signal = ac.signal;
+    ac.abort();
+    const [rli] = getInterface({ terminal });
+    assert.rejects(
+      rli.question('hello?', { signal }),
+      {
+        code: 'ABORT_ERR'
+      }
+    );
```

Comment:
```suggestion
    ).then(common.mustCall());
```

We should make sure `assert.rejects` resolve, otherwise the test could pass when we don't throw `AbortError` at all.